content/docs/tutorials/instrumenting_http_server_in_go.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Instrumenting HTTP server written in Go  
+sort_rank: 3
+---
+
+In this tutorial we will create a simple Golang HTTP server and instrumentation it by adding a counter
```

Comment:
Note from call:
Follow consistency, some places `Golang` is used and `Go` in other places. Same for `Prometheus` with capital `P`.